extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -709,11 +721,10 @@ angular.module('oppia').component('svgFilenameEditor', {
       };
 
       var drawQuadraticCurve = function() {
-        var size = ctrl.fabricjsOptions.size;
         var curve = new fabric.Path('M 40 40 Q 95, 100, 150, 40', {
```

Comment:
Is this the default path? If so, create a descriptive variable and use that.

Reply:
Done.